app/views/works/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<main class="row">
+<h3>Edit this <%= @work.category %></h3>
+<section>
+  <form class="edit_work" id="edit_work_<%= @work.id %>" action="/works/<%= @work.id %>" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="_method" value="patch"><input type="hidden" >
```

Comment:
Why are you doing this all with HTML instead of `form_for`?  

The `description` field isn't working and the form doesn't default to the work's category.